docs/assets/scss/_sidebar.scss

```
@@ -373,9 +355,11 @@
     font-size: 1.2rem;
   }
 
-  & .td-sidebar-nav__section.without-child > a.td-sidebar-link {
-    font-weight: 400;
-    
+  // Leaf pages
+  & .td-sidebar-nav__section.without-child a.td-sidebar-link {
+    font-family: "Qanelas Soft", "Open Sans", sans-serif, -apple-system, BlinkMacSystemFont, segoe ui, Roboto, apple color emoji, segoe ui emoji, segoe ui symbol;
```

Stylelint (17.14.0)

[error] Expected quotes around "segoe ui" (font-family-name-quotes)

(font-family-name-quotes)